### ISSUE TITLE
Roadmap 2020

### ISSUE THREAD

**Issue by szarnyasg** [ldbc/ldbc_snb_datagen_spark#215]
* [x] introduce valid delete operations – done, see [GRADES-NDA paper](http://ldbcouncil.org/sites/default/files/datagen-deletions-grades-nda-2020.pdf)
* [x] migrate Datagen to Spark
* [x] tune the distribution of deletes (ongoing on the `tuning` branch, #235)
  * [x] once this is done, tune the scale factors 
* [ ] rework paramgen (#83)
  * [ ] get it scale to SF1k+ for Interactive. As the parameter curation involves a lot of computation (estimating the sizes of intermediate join results), we need to move away from a naïve Python code.
  * [ ] add new BI queries (Q16-Q20)
* [x] support generating microbatches of inserts/deletes
* [x] ~add reaction type (enum) to `likes` edges~ WONTFIX

See also proposed refactoring and optimizations in #227

**Comment by szarnyasg** [comment]
This issue is done except for parameter generation. To be continued in #236
